crates/forge-runner/src/lib.rs

```diff
@@ -183,10 +184,28 @@ fn build_test_details(test_name: &str, sierra_program: &Program) -> TestDetails
     }
 }
 
-fn compile_sierra_to_casm(sierra_program: &Program) -> CairoProgram {
-    let metadata_config = MetadataComputationConfig::default();
-    let metadata = calc_metadata(sierra_program, metadata_config).unwrap();
-    cairo_lang_sierra_to_casm::compiler::compile(sierra_program, &metadata, true).unwrap()
+#[derive(Serialize, Deserialize)]
+pub struct AssembledProgramWithDebugInfo {
+    pub assembled_cairo_program: AssembledCairoProgramWithSerde,
+    pub debug_info: Vec<(usize, usize)>,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct AssembledCairoProgramWithSerde {
+    pub bytecode: Vec<BigInt>,
+    pub hints: Vec<(usize, Vec<Hint>)>,
+}
```

Comment:
Are those structs copied?

Reply:
Would be nice to make a PR to the compiler if so

Reply:
AssembledCairoProgram is copied (lack of serde) but AssembledCairoProgramWithDebugInfo is not